src/App.js

```diff
@@ -1,25 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code> src/App.js </code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <header className="App-header">Header</header>
+      <Calculator />
     </div>
   );
 }
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/158274828-814bdc08-68c9-4802-9064-ff663c87b4fb.png)

Therefore:

- [ ] Please, kindly make your `APP` component to be class-based.

That way, we'll meet this particular requirement.

Reply:
@Meltrust I believe this requirement is about the Calculator component. And If you look at the Calculator component it's already a class-based component.

https://github.com/awaisanwar544/math-magician/blob/a8649aaf77e925dca20f99ace8657abab1cc71be/src/components/Calculator.js#L7

Reply:
Hey @awaisanwar544, I hadn't had the opportunity to read your comment.  The requirement is about _all_ the components in the project because on the next milestones, you will be refactoring them into function-based.  Kindly remember, `App` is a component too.  Lots of students get confused about this one, so I can't blame you.

Since your project has been approved already, just kindly remember this.  Happy coding!